crates/uopool/src/canonical/sanity_check.rs

```diff
@@ -459,6 +493,7 @@ mod tests {
 
     use super::*;
 
+    #[ignore]
```

Comment:
https://github.com/Vid201/aa-bundler/actions/runs/5166205480/jobs/9306373395
This is stopping me merge this pr.

Reply:
It would be great to mock the case locally instead of relying on external source in test cases.

Reply:
Yeah, agree. I was thinking of removing this test case for now and revisiting it later. You can remove it in your PR.